.ci/scripts/test_int_util.sh

```diff
@@ -5,24 +5,31 @@
 # Library of common shell routines for PGE integration testing.
 #
 
+# Base temp directory to use for file staging
+DEFAULT_TMP_ROOT="/data/tmp"
+
 test_int_parse_args()
 {
     # Parse command line arguments and set global variables to be used in calling script
     #
     while [[ $# -gt 0 ]]; do
     case $1 in
       -h|--help)
-        echo "Usage: $(basename $0) [-h|--help] [-t|--tag <tag>] [--testdata <testdata .zip file>] [--runconfig <runconfig .yaml file>]"
+        echo "Usage: $(basename $0) [-h|--help] [-t|--tag <tag>] [-i|--input-data <zip file>] [-e|--expected-data <zip file>] [--runconfig <runconfig .yaml file>] [--temp-root <path>]"
         exit 0
         ;;
       -t|--tag)
```

Comment:
Although the arguments are pretty obvious, it might be useful to use comments to describe the arguments much at we do for our python code.